packages/generator/src/artifacts/css/static-css.ts

```diff
@@ -7,8 +7,12 @@ export const generateStaticCss = (ctx: Context, sheet?: Stylesheet) => {
 
   if (!sheet) {
     const { optimize = true, minify } = config
-    const output = engine.sheet.toCss({ optimize, minify })
-    void ctx.hooks.callHook('generator:css', 'static.css', output)
+    let output = engine.sheet.toCss({ optimize, minify })
+
+    if (ctx.hooks['cssgen:done']) {
+      output = ctx.hooks['cssgen:done']?.({ artifact: 'static', content: output }) ?? output
```

Comment:
```suggestion
      output = ctx.hooks['cssgen:done']({ artifact: 'static', content: output }) ?? output
```

We could also rename `output` to `css`